src/components/input/input.js

```diff
@@ -75,12 +75,12 @@ export const Input = ({
           name={name}
           disabled={disabled}
           defaultValue={defaultValue}
-          placeholder={usePlaceholder && label}
+          placeholder={usePlaceholder ? label : ""}
           onChange={onChange}
           required={required}
-          min={type === "number" && min}
-          max={type === "number" && max}
-          step={type === "number" && step}
+          min={type === "number" && min ? min : void 0}
+          max={type === "number" && max ? max : void 0}
+          step={type === "number" && step ? step : void 0}
```

Comment:
Doesn't this have the same effect as original code?

Reply:
Not really. We would pass a `boolean` if the condition would be `false` and therefore it will result in an error complaining that only `string/number/undefined` is a valid type.

Reply:
Ah, yes! My bad :)